bin/ch_core.c

```diff
@@ -410,6 +512,91 @@ void run_user_command(char *argv[], const char *initial_dir)
    Tf (0, "can't execve(2): %s", argv[0]);
 }
 
+/* Set up the fake-syscall seccomp(2) filter. This computes and installs a
```

Comment:
From the bottom of my grammar nerd heart, _**THANK YOU**_ for knowing the difference between "setup," "set-up," and "set up!"  🏆 🤣